addon/components/sortable-group.js

```diff
@@ -33,17 +33,10 @@ export default Component.extend({
   */
   items: computed(() => a()),
 
-  /**
-    Position for the first item.
-    If spacing is present, first item's position will have to change as well.
-    @property itemPosition
-    @type Number
-  */
-  itemPosition: computed(function() {
-    let direction = this.get('direction');
-
-    return this.get(`sortedItems.firstObject.${direction}`) - this.get('sortedItems.firstObject.spacing');
-  }).volatile(),
```

Comment:
I don’t remember why this property needed to be volatile. At first glance, it looks like we could just declare the dependent keys as `sortedItems.firstObject.{x,y,spacing}` but maybe I’m missing something.